nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -121,19 +122,24 @@ def _pooling(name):
     def _impl(inputs, attr, params):
 
         attr['data_format'] = attr['data_format'].decode("utf-8")
+        flip_layout = False
 
-        if attr['data_format'] == 'NHWC':
-            attr['kernel_shape'] = (attr['ksize'][1], attr['ksize'][2])
-        elif attr['data_format'] == 'NCHW':
-            attr['kernel_shape'] = (attr['ksize'][2], attr['ksize'][3])
-        else:
-            raise TypeError("Unsupported data_format type : {}".format(attr['data_format']))
+        input_shape = attr['_input_shapes'][inputs[0]][0]
+
+        if attr['_target_layout'] == "NCHW" and attr['data_format'] == "NHWC":
+            tmp_shape = attr['_input_shapes'][inputs[0]][0]
+            tmp_shape = [tmp_shape[ii] for ii in (0, 3, 1, 2)]
```

Comment:
input_shape = [tmp_shape[ii] for ii in (0, 3, 1, 2)]

Reply:
Done